tests_pytest/common_tests/unit_tests/test_filter_nodes_candidates.py

```diff
@@ -0,0 +1,247 @@
+# Copyright 2025 Sony Semiconductor Israel, Inc. All rights reserved.
```

Comment:
find the correct location of this test file according to the source code location of the tested function